File: ExcAlgoritmoProgramacao_JS/Lista_Bee/1012_basico.js
import { question } from "readline-sync";

function main() {
    const valores = get_number();

    const Nvalores = valores.split(` `);
    let A = parseFloat(Nvalores[0]);
    let B = parseFloat(Nvalores[1]);
    let C = parseFloat(Nvalores[2]);

    let triangulo = Area_Triangulo(A, C);
    let circulo = Area_Circulo(C);
    let trapezio = Area_Trapezio(A, B, C);
    let quadrado = Area_Quadrado(B);
    let retangulo = Area_Retangulo(A, B);

    print(`TRIANGULO: ${triangulo.toFixed(3)}`);
    print(`CIRCULO: ${circulo.toFixed(3)}`);
    print(`TRAPEZIO: ${trapezio.toFixed(3)}`);
    print(`QUADRADO: ${quadrado.toFixed(3)}`);
    print(`RETANGULO: ${retangulo.toFixed(3)}`);
}

function get_number() {
    return (question(`Numero: `));
}

function print(texto) {
    return console.log(texto);
}

function Area_Triangulo(A, C) {
    return (A * C) / 2;
}

function Area_Circulo(C) {
    let pi = 3.14159;

    return pi * (C ** 2);
}

function Area_Trapezio(A, B, C) {
    return ((A + B) * C) / 2;
}

function Area_Quadrado(B) {
    return B ** 2;
}

function Area_Retangulo(A, B) {
    return A * B;
}
main();

File: Exercicios_ED_I/Bee/1012_basico.cpp
#include <stdio.h>
#include <iostream>

using namespace std;

main() {
    double a, b, c;
    cin >> a >> b >> c;

    double area_TrianRetang = (a * c) / 2;
    double area_Circulo = 3.14159 * (c * c);
    double area_Trapezio = ((a + b) * c) / 2;
    double area_Quadrado = b * b;
    double area_Retangulo = a * b;

    printf("TRIANGULO: %.3lf\n", area_TrianRetang);
    printf("TRIANGULO: %.3lf\n", area_Circulo);
    printf("TRIANGULO: %.3lf\n", area_Trapezio);
    printf("TRIANGULO: %.3lf\n", area_Quadrado);
    printf("TRIANGULO: %.3lf\n", area_Retangulo);
}